cogs/vxtwitconverter/vxtwitconverter.py

```diff
@@ -0,0 +1,8 @@
+from cogs.vxtwitconverter.eventHandlers import EventHandlers
```

Comment:
Cog doesn't load, need the following changes:
```suggestion
from .eventHandlers import EventHandlers
```